### ISSUE TITLE
Emulator 4.6 fails to update to 4.7

### ISSUE THREAD

**Issue by brianrandell** [microsoft/BotFramework-Emulator#2096]
After downloading the update via the emulator, the bell icon lights up offering dismiss or restart. Restart does not work. 

Closing manually does not cause an update to run. 

Multiple tries ends in error dialogs like the attached screen shot.

![image](https://user-images.githubusercontent.com/1954458/76348421-67eebd00-62c5-11ea-9664-7e757b02ab64.png)

![image](https://user-images.githubusercontent.com/1954458/76348434-6de49e00-62c5-11ea-9996-fdf1e28f5e9f.png)

**Comment by tonyanziano** [comment]
Hi @brianrandell ,

This is a known issue (#1667).

Unfortunately when we have tried to fix this we have not been able to reproduce the issue which makes it very hard to debug. When you say you tried to close manually, did you try closing the Emulator immediately after the update has been downloaded and without pressing Dismiss / Restart?

It might also be worth trying to check for an update manually via Help -> Check for update...

![image](https://user-images.githubusercontent.com/3452012/76437095-4a2d6080-6376-11ea-90e2-9de5f4c17c6f.png)

and then choosing "Install this update and restart Emulator"

![image](https://user-images.githubusercontent.com/3452012/76437195-6cbf7980-6376-11ea-9def-ca632ab4c5eb.png)

**Comment by brianrandell** [comment]
@tonyanziano thanks. I tried multiple "different" approaches. I can try again. I never got image two with the "update available" dialog. I'll report back later today. Thanks!

**Comment by brianrandell** [comment]
TL;DR;
I was able to get it to update tonight! 

**Details**

Started Emulator.
Checked Help | About
	4.6.0

Bell lights up while I type this message.
Click the Bell.

![image](https://user-images.githubusercontent.com/1954458/76721906-d33df200-66fe-11ea-8a6a-4965f584ca1f.png)

Click **Restart** and nothing happens.
Click **Restart** again and nothing happens.
Click **Dismiss** and message goes away.
Exit Emulator, nothing happens.

Restart Emulator
Go to Help | Check for update.

Receive the following dialog twice.
![image](https://user-images.githubusercontent.com/1954458/76721921-df29b400-66fe-11ea-8a74-42b2a0f2d1ad.png)

Exit Emulator.
Restart Emulator.
Checking Help menu this time shows checking for update and then an option to Restart.

I select it. 

The Emulator appears to minimize. 

This is what Task Manager shows.
![image](https://user-images.githubusercontent.com/1954458/76721937-ea7cdf80-66fe-11ea-8f5f-d524ff4d6ca9.png)



I click minimized "app" and then it restores to an installer:
![image](https://user-images.githubusercontent.com/1954458/76721940-eea8fd00-66fe-11ea-9d12-711be9288f3c.png)



Click Next, completes.
![image](https://user-images.githubusercontent.com/1954458/76721943-f1a3ed80-66fe-11ea-8ee6-d6d92c227fbb.png)

**Comment by brianrandell** [comment]
Well, I didn't really mean to close it. Had keyboard issue. But at this point it works. Let me know if I can help any more on this.

**Comment by tonyanziano** [comment]
@brianrandell Glad you got it working, and thanks a lot for the detailed step-by-step report!

That error dialog makes me think it might have something to do with some process in the installer not being elevated to the correct privileges on your machine. I wonder if trying to do this while running the Emulator as admin would have helped.

**Comment by brianrandell** [comment]
Yeah, interesting. I was logged in as my MSA which is a local admin but I have UAC on with Windows 10. No UAC dialogs were presented.

**Comment by congysu** [comment]
thanks @brianrandell for reporting the issue.   we'll close this issue for now, and feel free to reopen it if needed.
